Src/Test.UnitTests.Sarif.PatternMatcher/SearchSkimmerTests.cs

```diff
@@ -447,12 +442,8 @@ public void SearchSkimmer_ShouldNotEvaluateTooLargeFiles()
                     maxFileSize = int.MaxValue,
                 },
                 new {
-                    fileSize = (long)ulong.MinValue,
-                    maxFileSize = (int)0,
-                },
-                new {
-                    fileSize = (long)ulong.MinValue,
-                    maxFileSize = int.MaxValue,
+                    fileSize = (long)50000000,
+                    maxFileSize = (int)10000,
```

Comment:
For the test that verifies too large files  are not scanned.  Missing testcase of a fileSize that is default max + 1 which not explicitlly setting the max (i.e. using the default value).  Not seeing that case anywhere

Reply:
Added an additional test to cover this and included int.MaxValue as an additional test case. See `SearchSkimmer_SearchSkimmer_ShouldNotEvaluateFilesExceedingDefaultLimit`